### PR TITLE
Added packetlight support

### PR DESCRIPTION
**Pull request by PipoCanaja**
Packetlight support, using 2 simple user_func for the substraction of 30 and 128 resp, included into the Packetlight class. Standard YAML :)

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
How about putting the functions in an OS class `\LibreNMS\OS\Packetlight`?  Then put them in yaml like `LibreNMS\OS\Packetlight::offsetDbm`

This way we don't have random functions in the general php and know what OS they are for.

**Comment by PipoCanaja** [comment]
Yep, I like that. I'll update the PR accordingly.

File: includes/discovery/functions.inc.php [hunk 1]
```diff
@@ -1107,7 +1107,7 @@ function discovery_process(&$valid, $device, $sensor_type, $pre_cache)
                     }
 
                     //user_func must be applied after divisor/multiplier
-                    if (isset($user_function) && function_exists($user_function)) {
+                    if (isset($user_function) && is_callable($user_function)) {
```
Comment (PipoCanaja) [inline includes/discovery/functions.inc.php]: @murrant I had to change this function_exists to is_callable to handle the 'class' that comes with the name. Looks safe from the documentation.

**Comment by murrant** [comment]
@PipoCanaja thanks for working through this.  I'm pretty happy with how it came out.

**Comment by PipoCanaja** [comment]
I learnt a couple of interesting things doing it :) Let see how Travis likes it.

**Pull request merged**